plugins/tracker-resources/src/components/issues/IssueNotification.svelte

```diff
@@ -71,8 +82,26 @@
     </div>
   </svelte:fragment>
 
-  <svelte:fragment slot="buttons">
+  <svelte:fragment slot="buttons"  >
     <Button label={tracker.string.ViewIssue} on:click={handleIssueOpened} />
     <Button icon={view.icon.CopyLink} label={tracker.string.CopyIssueUrl} on:click={handleCopyUrl} />
+
+    {#if showCopiedTooltip}
+      <div class="tooltip">Copied URL!</div>
```

Comment:
String should be internationalised.

Reply:
Hi, I am not able to get the value internationalised, could you please help me out in this?

Reply:
@haiodo